lib/Agrammon/OutputFormatter/Text.pm6

```diff
@@ -84,6 +88,13 @@ sub render-filters(@module-lines, Agrammon::Outputs::FilterGroupCollection $coll
     }
 }
 
+multi sub flat-value($value) {
+    $value
+}
+multi sub flat-value(Agrammon::Outputs::FilterGroupCollection $collection) {
+    +$collection
+}
+
```

Comment:
I do wonder if we should put this `and sorted-kv` into some kind of `Agrammon::OutputUtil` module so we can re-use them...

Reply:
Done.